src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -32,22 +32,17 @@ import scalariform.ScalaVersion
 import scala.collection.JavaConverters._
 import scala.util.{Failure, Success, Try}
 
-/** Main sensor for importing Scoverage reports to SonarQube */
-final class ScoverageSensor extends ScoverageSensorInternal with ScoverageReportParser
-
 /** Implementation of the sensor */
-private[scoverage] abstract class ScoverageSensorInternal extends Sensor {
-  // cake pattern to mock the scoverage report parser in tests
-  scoverageReportParser: ScoverageReportParserAPI =>
+final class ScoverageSensor(scoverageReportParser: ScoverageReportParserAPI) extends Sensor {
 
-  private[this] val log = Log(classOf[ScoverageSensorInternal], "scoverage")
+  private val log = Log(classOf[ScoverageSensor], "scoverage")
```

Comment:
I'm just curios about why removing the `[this]` private modifier. In theory not even another instance should be able to access the logger of this instance.

Reply:
Yeah, you're right!